### PR TITLE
Must be install dir, not GRAALVM_HOME

### PR DESCRIPTION
**Pull request by ebullient**

### CODE REVIEW

File: docs/src/main/asciidoc/building-native-image-guide.adoc [hunk 1]
```diff
@@ -110,7 +110,7 @@ Use the following command to recursively delete the `com.apple.quarantine` exten
 
 [source,shell]
 -----
-xattr -r -d com.apple.quarantine ${GRAALVM_HOME}
+xattr -r -d com.apple.quarantine $HOME/Development/graalvm/
```
Comment (stuartwdouglas) [inline docs/src/main/asciidoc/building-native-image-guide.adoc]: What is this directory? Is this basically just your workspace dir?

Comment (gsmet) [reply docs/src/main/asciidoc/building-native-image-guide.adoc]: Maybe let's make it `${GRAALVM_INSTALL_DIRECTORY}`

Comment (ebullient) [reply docs/src/main/asciidoc/building-native-image-guide.adoc]: It is the directory of the install, which is referenced earlier in the document.

> Configure the runtime environment. Set GRAALVM_HOME environment variable to the GraalVM installation directory, for example:
> export GRAALVM_HOME=$HOME/Development/graalvm/
> On macOS, point the variable to the Home sub-directory:
> export GRAALVM_HOME=$HOME/Development/graalvm/Contents/Home/

If we want to add an explicit set of GRAALVM_INSTALL_DIR at this point, we could do that (and then reference it in this later section). But I reused the string in common with the install instructions.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
OK, let's merge, it's probably clear enough.

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**